test/controllers/LookupControllerSpec.scala

```diff
@@ -150,11 +158,20 @@ class LookupControllerSpec extends FakeApplication with BeforeAndAfterEach with
     }
 
     "return 200 with correct message on protection type guidance page" in new Setup {
+      when(mockAppConfig.psalookupjourneyShutterEnabled).thenReturn(false)
+      val request = FakeRequest().withSession(sessionId)
+      val result  = controller.displayProtectionTypeGuidance.apply(request)
+
+      status(result) shouldBe OK
```

Comment:
These test don't test which view/template is being displayed. You can see an example how to do it here: https://github.com/hmrc/pensions-lifetime-allowance-frontend/pull/559/files#diff-c8a5e6bcb4ed99c6f585866495531a35ec667595a0dd7ea1697e391c0524fe31